python/lsst/cell_coadds/_fits.py

```diff
@@ -19,14 +19,57 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+"""Module to handle FITS serialization and de-serialization.
+
+The routines to write and read the files are in the same module, as change to
+one is typically accompanied by a corresponding change to another.
+Changes to this module may require a bump to the version number denoted by the
+module constant FILE_FORMAT_VERSION. The policy for bumping the version is:
+
+1. When the on-disk format written by this module changes such that the
+accompanying reader can still read files written by the previous version, then
```

Comment:
I think minor bumps are when older readers can still read files written by new writers.  As you've described it here, the _format_ is not changing; only the reader code is.  If you don't want to try to rigorously support compatibility in that direction (as suggested in the examples), then we could say "old readers _may_ be able to read new code", while major would mean they definitely cannot.